config/config.example.json

```diff
@@ -12,8 +12,5 @@
   "timeout": 10,
```

Comment:
Good commit! The project is configured with a bunch of tools that help make the code more consistent, glad that you used it :)